### ISSUE TITLE
Plugin Details: Tested Up To version has the wrong version

### ISSUE THREAD

**Issue by cpapazoglou** [Automattic/wp-calypso#58283]
### Details

Tested Up To refers to the version of WordPress that that plugin has been tested with. On plugin detail pages we seem to show the plugin version. I might have mentioned that in my feedback to the designs already, but I wonder if this information is even needed, given that customers can’t change their WordPress version? It doesn’t feel to me like like it adds much value.

![image](https://user-images.githubusercontent.com/12430020/142630728-65f7d4c7-cdc7-4b28-891d-94534488f6d9.png)

### Related
pdh6GB-6h-p2#comment-315

**Comment by cpapazoglou** [comment]
@obenland I agree that this information doesn't add any value. What if we rephrased that to "Not tested with the latest WordPress version" as a warning when applicable?

**Comment by obenland** [comment]
I'd anticipate that the vast majority of plugins fall under that. Not being tested with the latest version of WP doesn't necessarily mean that they're incompatible or even broken given WordPress' emphasis on backwards compatibility. 

So I'd examine what we actually want to convey with that information and whether we're just showing it because it's available.

**Comment by cpapazoglou** [comment]
> I'd anticipate that the vast majority of plugins fall under that. Not being tested with the latest version of WP doesn't necessarily mean that they're incompatible or even broken given WordPress' emphasis on backwards compatibility.
> 
> So I'd examine what we actually want to convey with that information and whether we're just showing it because it's available.

I would suggest keep showing the "Tested up to":

- keeping parity with WP core plugin details screen
- we also have to deal with Jetpack sites with different WordPress versions

**Comment by obenland** [comment]
> keeping parity with WP core plugin details screen

Can you tell me more about that goal and its rationale? Is there prior discussion around this?

**Comment by cpapazoglou** [comment]
> > keeping parity with WP core plugin details screen
> 
> Can you tell me more about that goal and its rationale? 

Since we have forced users to use Calypso instead of WP Admin we got user reports that the new interfaces are lacking behind the WP Admin interfaces. The rationale of bringing some parity is not to frustrate users regarding the changes right before we launch the paid plugins (which they will further "bloat" the Plugins Browser).

> Is there prior discussion around this?

Yes, here is an issue https://github.com/Automattic/wp-calypso/issues/58319 where we discuss parity between Calypso & WP Admin.

We can discuss it further though! I just thought, that it might be a good idea of displaying "Tested Up To" only for Jetpack sites?
